pkg/daemon/update.go

```diff
@@ -59,6 +59,13 @@ const (
 	// GPGNoRebootPath is the path MCO expects will contain GPG key updates. MCO will attempt to only reload crio for
 	// changes to this path. Note that other files added to the parent directory will not be handled specially
 	GPGNoRebootPath = "/etc/machine-config-daemon/no-reboot/containers-gpg.pub"
+
+	// excludeFromLoadBalancerLabel is used by the cloud service controller in order to know
+	// which nodes to exclude from backend pools of cloud load balancers.
```

Comment:
Since you moved your funcs to daemon.go, these consts are orphaned. can we move them to live in daemon.go?

Reply:
Yes, absolutely!